app/models/manageiq/providers/redfish/manager_mixin.rb

```diff
@@ -17,6 +17,12 @@ def connect(options = {})
     self.class.raw_connect(username, password, host, port, protocol)
   end
 
+  def disconnect(connection)
+    connection.logout
+  rescue StandardError => error
```

Comment:
I tend to disagree with rubocop on this one (I assume that is why you did this :smile:), I don't think there is anything wrong with `rescue => err`.
Totally up to you but if you want to come back and clean this up if you prefer `rescue => error` I'm good with either one.